packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8630,34 +8652,6 @@ describe('plugin-meetings', () => {
 
           checkParseMeetingInfo(expectedInfoToParse);
         });
-
-        it('should parse meeting info, set values, and return null when permissionToken is not present', () => {
```

Comment:
why is this unrelated test being removed as part of this PR? is this some merge gone wrong?

Reply:
I didn't remove this? I'll see what happened here

Reply:
Ok I must've command + z'd this test out of the file. Good catch!